libclamav/pdf.c

```diff
@@ -1440,6 +1440,11 @@ cl_error_t pdf_extract_obj(struct pdf_struct *pdf, struct pdf_obj *obj, uint32_t
 
     cli_dbgmsg("pdf_extract_obj: obj %u %u\n", obj->id >> 8, obj->id & 0xff);
 
+    if (PDF_OBJECT_RECURSION_LIMIT < rec_level) {
```

Comment:
That's a very non-intuitive comparison.  I would've done
``` c
if (rec_level >= PDF_OBJECT_RECURSION_LIMIT)
```

Reply:
Yeah I write a lot of [yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions).  I'm not ashamed 😅. Every now and then it'll catch a bug.